website/sidebars.js

```diff
@@ -31,61 +42,112 @@ const sidebarSettings = {
               ],
             },
             {
               type: "category",
-              label: "Learning more",
+              label: "Manage access",
+              items: [
+                "docs/cloud/manage-access/about-access",
+                "docs/cloud/manage-access/seats-and-users",
+                {
+                  type: "category",
+                  label: "Permissions",
+                  items: [
+                    "docs/cloud/manage-access/self-service-permissions",
+                    "docs/cloud/manage-access/enterprise-permissions",
+                  ],
+                },
+    
+                {
+                  type: "category",
+                  label: "Single sign-on",
+                  items: [
+                    "docs/cloud/manage-access/sso-overview",
+                    "docs/cloud/manage-access/set-up-sso-saml-2.0",
+                    "docs/cloud/manage-access/set-up-sso-okta",
+                    "docs/cloud/manage-access/set-up-sso-google-workspace",
+                    "docs/cloud/manage-access/set-up-sso-azure-active-directory",
+                  ],
+                }, // SSO
+                {
+                  type: "category",
+                  label: "OAuth with data platforms",
+                  items: [
+                    "docs/cloud/manage-access/set-up-snowflake-oauth",
+                    "docs/cloud/manage-access/set-up-bigquery-oauth",
+                  ],
+                }, // oauth
+                "docs/cloud/manage-access/audit-log",
+              ],
+            }, // Manage access
+    
+            {
+              type: "category",
+              label: "Configure Git",
+              items: [
+                "docs/cloud/git/connect-github",
+                "docs/cloud/git/connect-gitlab",
+                {
+                  type: "category",
+                  label: "Azure DevOps",
+                  items: [
+                    "docs/cloud/git/connect-azure-devops",
+                    "docs/cloud/git/setup-azure",
+                    "docs/cloud/git/authenticate-azure",
+                  ],
+                },
+                "docs/cloud/git/import-a-project-by-git-url",
+              ],
+            }, // Supported Git providers
+            
+            {
+              type: "category",
+              label: "Multi-tenant PrivateLink",
               items: [
-                "docs/get-started/learning-more/using-jinja",
-                "docs/get-started/learning-more/refactoring-legacy-sql",
+                "docs/cloud/privatelink/about-privatelink",
+                "docs/cloud/privatelink/snowflake-privatelink",
+                "docs/cloud/privatelink/redshift-privatelink",
+                "docs/cloud/privatelink/databricks-privatelink",
               ],
-            },
+            }, // PrivateLink
+            "docs/cloud/develop-in-the-cloud",
+            "docs/cloud/dbt-cloud-tips",
```

Comment:
ooc, should these two Cloud pages be under the new "About dbt Cloud" top-level?

Reply:
I split the "develop in the cloud" into two docs so there is an "About Cloud IDE" section in the "About Cloud" folder but the dbt cloud tips is a bit of a wildcard. It's functional, so I'm not sure it belongs in the About Cloud folder but I'm open to ideas if this doesn't seem like the right place